SPECS/nlopt/nlopt.spec

```diff
@@ -0,0 +1,505 @@
+# Prevent accidental soname bumps.
+%global sover        0
+# Conditionals controlling the build.
+%global with_guile   0
+%global with_octave  0
+%global with_py3     0
```

Comment:
Do we plan on enabling any of these? I'd be supportive of removing what's not used.

Reply:
I think they are properly gated and I am of the opinion that it would be trivial effort just in case we want to enable them latter if a business case arise @oliviacrain

Reply:
Sure thing, I trust your judgement here.